lib/api/user_service.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:dio/dio.dart';
+import 'package:flutter_survey/api/response/user_response.dart';
+import 'package:retrofit/retrofit.dart';
+
+part 'user_service.g.dart';
+
+class SurveyApi {
+  SurveyApi._();
+}
```

Comment:
is this class redundant? I see it's unused 💭

Reply:
oops, I was copied it from `survey_service`. Fixed https://github.com/luongvo/flutter-survey/pull/69/commits/1b0740e2062897e2cef055213be24ca43b84114c 😂